### PR TITLE
Display signs images

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

**Comment by yCobanoglu** [comment]
[Display annotations in Signs interface](https://trello.com/c/C3BG2qRF/538-display-annotations-in-signs-interface)

File: src/signs/ui/display/__snapshots__/SignDisplay.test.tsx.snap [hunk 1]
```diff
@@ -2,488 +2,24 @@
...
-    >
-      <nav
-        aria-label="breadcrumb"
+    <h4>
+      Something's gone wrong.
+    </h4>
+    <p>
+      Our team has been notified, but you can fill out a report by clicking the button below.
+    </p>
+    <p>
```
Comment (jlaasonen) [inline src/signs/ui/display/__snapshots__/SignDisplay.test.tsx.snap:15]: Fix the test.

File: src/signs/ui/display/SignImages.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    </Row>
+  )
+
+  return (
+    <Container>
+      <SignImagePage chunk={chunks[activePage - 1]} />
+      <Row>
+        <Col xs={{ offset: 5 }}>
+          {' '}
+          <Pagination>{items}</Pagination>
```
Comment (jlaasonen) [inline src/signs/ui/display/SignImages.tsx:82]: Wouldn't it be better to have to create all the pagination components in the same place? You could extract a component for the pagination.

**Review by jlaasonen — CHANGES_REQUESTED** [review]

File: src/signs/ui/display/SignImages.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.SignImages__signImage {
```
Comment (jlaasonen) [inline src/signs/ui/display/SignImages.css:1]: Use lower case names for CSS classes: https://github.com/ElectronicBabylonianLiterature/ebl-frontend#htmlcss

If I remember correctly, we discussed this at some point.

**Review by jlaasonen — CHANGES_REQUESTED** [review]

File: src/signs/ui/display/SignImages.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.signImages__signImage {
```
Comment (jlaasonen) [inline src/signs/ui/display/SignImages.css:1]: Change all letters to lower case.

**Review by jlaasonen — CHANGES_REQUESTED** [review]

**Pull request merged**